client/basic-authorship/src/lib.rs

```diff
@@ -54,5 +54,6 @@
 //!
 
 mod basic_authorship;
+mod metrics;
```

Comment:
Can we make this reusable? I don't want to have this copy the code for polkadot. Maybe in its own crate?

Reply:
Ok!

Reply:
@bkchr moved to crate separate